app/helpers/clubs_helper.rb

```diff
@@ -1,4 +1,9 @@
 module ClubsHelper
+  def club_previous_attendee_count(club)
+    previous_meeting = club.meetings.where(["created_at < ?", Date.today]).last
+    previous_meeting.blank? ? 'N/A' : previous_meeting.feedback_responses.length
+  end
```

Comment:
Another option may be to omit this helper entirely and putting something along
the lines of the following straight into the view:

```haml
- if @current_club and @current_club.meetings.count > 0
  = @current_club.meetings.last.attendees
```

And then not display anything for attendance if the member isn't part of a club
yet or if there haven't been any meetings.